docs/visual_parity.md

```diff
@@ -0,0 +1,56 @@
+
+## Using VisualParity in CI/CD
+
+Running VisualParity tests in CI/CD platforms allows the automated verification that web page matches the baseline images. This can be particularly helpful as a pre-deployment action to verify that no unexpected frontend changes have occurred before deploying chanegs.
+
+### LocalStorage - Checking In Images
+
+Many cloud CI/CD systems use ephemeral (or effectively ephemeral) runners, which do not save state between runs, or clean up the file system between runs. In these cases, a failed test will give a locator to a file that may no longer exist on the system. However, most CI/CD systems support producing artifacts. Therefore, LocalStorage can be used while still making the run images available by exporting the runs directory as an artifact.
+
+The workflow of using this is as follows:
+
+1. Run `--update-all-baselines` to generate the necessary baseline images and commit them to the repository
+1. Run the Quilla tests through the CI system
+1. If a test failed, upload the `<image_directory>/runs` folder as an artifact
+1. Download the artifact, compare the failed images
+1. If baseline images need to be updated, run `--update-baseline <baseline_id>` and commit the new baseline image to the repository
+
+### BlobStorage - Storing Baselines in the Cloud
+
+Azure Blob Storage containers can be used to store both the baseline images as well as treatment images. Using the `--connection-string` CLI option, a connection string can be passed that gives Quilla access to a specific storage account. Optionally, a container name can also be given with the `--container-name` option, but Quilla will by default use a container called "quilla", creating it if it does not exist.
+
+> **Note**: The Azure Blob Storage connection string contains an actual key and **MUST NEVER** be committed to the repository, especially on public repositories. CI systems (such as Github Actions, Azure DevOps, etc.) have secret management systems such as key vaults that are designed to store private information and should be used instead.
+
+The workflow for using Azure Blob Storage with Quilla in CI/CD is as follows:
+
+1. Run `--update-all-baselines` to generate the baselines and upload them to Blob Storage
+1. Run Quilla tests through the CI system
+1. If a test failed, the test report will include a URL to the treatment and baseline image. Depending on how the storage account is configured, this link might work directly. If the permissions for the storage account do not allow for public view access, the images will need to be downloaded directly from Azure Blob Storage.
+1. If baseline images need to be updated, run `--update-baseline <baseline_id>`, which will generate the new baselines and upload them to Blob Storage.
+
```

Comment:
Any prod usage of Quilla VP feature will have to go through an online storage. Could you also add notes that both `--update-all-baselines` and `--update-baseline <baseline_id>` can be triggered through user's own CI system (and why Quilla choose to stay simple here)

Reply:
Yeah, prod usage of Quilla VP features I think would count as a "post-deploy" which I have as a section at the bottom of this doc.

I'll add that section on updating baselines with CI/CD  but I'll do so with the caveat that the generally expected use is for the baseline updates to happen through the CLI